src/signer.ts

```diff
@@ -12,6 +12,14 @@ export abstract class Signer {
      */
     abstract async createKey(accountId: string, networkId?: string): Promise<PublicKey>;
 
+    /**
+     * Creates a public key for the account, network and keyPair provided
+     * @param accountId accountId to retrieve from.
+     * @param networkId The targeted network. (ex. default, betanet, etc…)
+     * @param keyPair The keyPair to use for signing
+     */
+    abstract async fromKeyPair(accountId: string, networkId: string, keyPair: KeyPair): Promise<PublicKey>;
```

Comment:
@mattlockyer let's keep the same order of `networkId` / `accountId` as in `setKey` to avoid additional confusion